lib/ui/src/settings/index.tsx

```diff
@@ -85,7 +84,6 @@ const Pages: FunctionComponent<{
         <ShortcutsPage key="shortcuts" />
       </Route>
     </Content>
-    <GlobalHotKeys handlers={{ CLOSE: onClose }} keyMap={keyMap} />
```

Comment:
What's the replacement for this?

Reply:
I don't know. I just wanted to see what the perf gain would be on the benchmark.